boards/common/xsdb.board.cmake

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2024 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+board_set_debugger_ifnset(xsdb)
+board_set_flasher_ifnset(xsdb)
+board_finalize_runner_args(xsdb)
+set(FLASH_RUNNER xsdb)
```

Comment:
What is the last line here doing?

Reply:
it's also a unnecessary line removed now thanks for the quick review and reply.